root/static/scripts/edit/components/ArtistCreditBubble.js

```diff
@@ -281,6 +280,7 @@ component _ArtistCreditBubble(
         <tbody>
           {namesWithInitialFocus.map((name, index) => (
             <ArtistCreditNameEditor
+              allowMove={allowNameRemoval && !name.removed}
```

Comment:
Is it reasonable to just use the `allowNameRemoval` condition here?

Reply:
That looks fine; we can change it to `allowNameMoveOrRemoval` if you prefer? (`allowNameMoval`? 😆)

Reply:
Thanks, I'll push an update using the (first :-P) name after Jenkin reports the inevitable test mistakes that I'll need to fix.